lib/pbench/test/unit/server/query_apis/test_datasets_detail.py

```diff
@@ -323,7 +329,7 @@ def test_nonunique_query(
         Check the handling of a query that returns too much data.
         """
         json = {
-            "user": "drb",
+            "user": "",
             "name": "fio",
             "start": "2020-08",
             "end": "2020-10",
```

Comment:
Since the mock provides the response, how much of a realistic payload do we actually need?

Reply:
They need to be realistic, we have validator check on each of these json fields and if they don't match with how its define in our code then we respond back as a bad request

Reply:
[Answered](https://github.com/distributed-system-analysis/pbench/pull/2251#discussion_r656372613).